### PR TITLE
Add IVetoProjectLoad example

### PR DESCRIPTION
**Pull request by jviau**

### CODE REVIEW

File: doc/extensibility/IVetoProjectLoad.md [hunk 1]
```diff
@@ -3,4 +3,24 @@
 
 **[Item template:](project_item_templates.md)** Project Load Veto extension
 
-TODO
+To veto a project load there is two extension points `IVetoProjectLoad` and `IVetoProjectPreLoad`.
```
Comment (adrianvmsft) [inline doc/extensibility/IVetoProjectLoad.md]: May be useful to specify what "veto a project load means" - e.g. prevent the project to load

File: doc/extensibility/IVetoProjectLoad.md [hunk 2]
```diff
@@ -3,4 +3,24 @@
...
+The difference is that `IVetoProjectPreLoad` is called with the configuration about to be loaded.
+
+Example implementation:
```
Comment (adrianvmsft) [inline doc/extensibility/IVetoProjectLoad.md]: A quick summary of what the sample does may be usefull

Comment (jviau) [inline doc/extensibility/IVetoProjectLoad.md]: I think the code is self explanatory. `AllowProjectLoadAsync` is a very descriptive name.

File: doc/extensibility/IVetoProjectLoad.md (hunk 1 above)
Comment (jviau) [inline doc/extensibility/IVetoProjectLoad.md]: To veto something means to stop something already, no need to repeat that.

**Pull request merged**